Celeste.Mod.mm/Content/Dialog/Polish.txt

```diff
@@ -71,6 +77,19 @@
 	MODOPTIONS_COREMODULE_INPUTGUI_PS4=						PS4
 	MODOPTIONS_COREMODULE_INPUTGUI_XB1=						XBONE
 	MODOPTIONS_COREMODULE_INPUTGUI_TOUCH=					TOUCH
+	MODOPTIONS_COREMODULE_COMPATMODE=				Tryb Kompatybilności
+	MODOPTIONS_COREMODULE_COMPATMODE_NONE=				WYŁ.
+	MODOPTIONS_COREMODULE_COMPATMODE_NONE_DESCR_A=
+	MODOPTIONS_COREMODULE_COMPATMODE_NONE_DESCR_B=
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYXNA=				STARE XNA
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYXNA_DESCR_A=		Przywraca zachowanie starych wersji XNA, pozwalając na częstotliwość 61 klatek na sekundę
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYXNA_DESCR_B=		PRZEZNACZONE TYLKO DLA UŻYTKOWNIKÓW DOTKNIĘTYCH PRZEZ TĄ USTERKĘ!
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYFNA=				STARE FNA
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYFNA_DESCR_A=		Przywraca zachowanie starych wersji FNA, polegające na dodatkowym opóźnieniu sterowania
+	MODOPTIONS_COREMODULE_COMPATMODE_LEGACYFNA_DESCR_B=		PRZEZNACZONE TYLKO DLA UŻYTKOWNIKÓW PRZYZWYCZAJONYCH DO TEGO ZACHOWANIA!
+	MODOPTIONS_COREMODULE_COMPATMODE_INCOMPATIBLE=			Uwaga: Brak kompatybilności z oprogramowaniem twojej niemodyfikowanej instalacji!
+	MODOPTIONS_COREMODULE_D3D11EXCLUSIVEFULLSCREEN=			Tryb Ekskluzywnego Pełnego Ekranu
+	MODOPTIONS_COREMODULE_D3D11EXCLUSIVEFULLSCREEN_DESC=		Ma efekt na renderowaniu przy użyciu D3D11 na systemach Windows
```

Comment:
```suggestion
	MODOPTIONS_COREMODULE_D3D11EXCLUSIVEFULLSCREEN_DESC=		Dotyczy renderera D3D11 na systemach Windows
```
This is the literal translation of its `English.txt` counterpart.

Reply:
oof, looking back at that line, i botched it really hard. thanks for this